apps/src/netsim/NetSimRouterNode.js

```diff
@@ -1203,14 +1203,17 @@ NetSimRouterNode.prototype.getNodeIDForAddress_ = function (address) {
 };
 
 /**
- * Given a network address, finds the node ID of the node that is the next
- * step along the shortest path from this router to that address.  Will return
- * undefined if no path to the address is found.
+ * Given a network address, finds the node that is the next step along the
+ * correct path from this router to that address.  Will return null if no
+ * path to the address is found.
```

Comment:
Some of the comment updates are in response to a change that was made a while back, to make this method return `NetSimNode` and not a `number` node ID.